mysql/resultset.go

```diff
@@ -39,7 +39,7 @@ func (p RowData) ParseText(f []*Field) ([]interface{}, error) {
 		if isNull {
 			data[i] = nil
 		} else {
-			isUnsigned = (f[i].Flag&UNSIGNED_FLAG > 0)
+			isUnsigned := f[i].Flag & UNSIGNED_FLAG != 0
```

Comment:
small style change--advantage being that the var isUnsigned can't be read without having being properly initialized